src/buffer/out/TextAttributeRun.hpp

```diff
@@ -25,20 +25,24 @@ Revision History:
 class TextAttributeRun final
 {
 public:
-    TextAttributeRun() noexcept;
-    TextAttributeRun(const size_t cchLength, const TextAttribute attr) noexcept;
+    TextAttributeRun() = default;
+    TextAttributeRun(const unsigned int cchLength, const TextAttribute attr) noexcept :
+        _cchLength(cchLength)
+    {

```

Comment:
For inlining? Where's the delete of the cpp copy?

Reply:
Right - when something is trivial, doing it in the header means it's a lot more likely to inline trivially.  LTCG can probably inline this later, and PGO surely would if it's hot enough, but it seems better to let the compiler do it, rather than playing chicken with the various optimization passes.  I think trivial getters and things like that should always be in headers for this reason.

For noexcept things, this also means it's more likely to skip the implicit try/catch that noexcept generates, if the compiler can prove nothing in the implementation can ever actually throw.

Reply:
Oh, and as to the cpp copy question - I thought I deleted all of TextAttributeRun.cpp - but maybe I messed up the deletion somehow?

Reply:
I just searched for the deletion of the file and didn't see it in the PR view. And I'm good with that... but it might be better to do it as a broad policy across all buffer output files in a separate PR instead of just an opportune one or two packed into this guy.

Reply:
I had deleted the file from the vcxproj but forgot to actually delete TextAttributeRun.cpp on-disk.  That's done in the latest revision of the PR.

Reply:
Mind updating the `sources` file for this directory as well? I'll handle the other stuff (adding the boost dependency et al)

Reply:
sources file is updated - and thanks for taking care of the boost part.